src/index.js

```diff
@@ -40,6 +40,57 @@ const _views = new WeakMap();
 
 let bypass = false;
 
+function nextOrPreviousProject (index) {
+    // Generate an array of projects by selecting children of 'project-viewer > ul[is="pv-list-tree"]'. For each 'li[is="pv-list-nested-item"]', expand it into its children.
+    const sidebar = document.getElementsByTagName('project-viewer')[0];
+    let list = sidebar.getElementsByTagName('ul')[0].children;
+
+    var output = [];
+
+    var expandProjects = function (element) {
+        if (element.getAttribute('is') === "pv-list-nested-item") {
+            let nestedList = element.getElementsByTagName('ul')[0].children;
+
+            for (var i = 0; i < nestedList.length; i++) {
+                expandProjects(nestedList[i]);
+            }
+        }
+        else if (element.getAttribute('is') === "pv-list-item") {
+            output.push(element);
+        }
+    };
+
+    for (var i = 0; i < list.length; i++) {
+        expandProjects(list[i]);
+    }
+
+    for (var i = 0; i < output.length; i++) {
+        var txt = txt + output[i].children[0].innerHTML + "\n";
+    }
+
+    // Identify the target project and simulate a click event.
```

Comment:
i have updated the db file so you can use `_utility.getDB().views.projects` and it will list you all the ordered projects so you can clean up the house with the for loops, is it something you are ok with?